test/docker/compose.go

```diff
@@ -117,20 +116,7 @@ type Compose struct {
 }
 
 func New() *Compose {
-	return &Compose{enableModules: []string{}, weaviateEnvs: make(map[string]string)}
-}
-
-func (d *Compose) WithMinIO() *Compose {
-	d.withMinIO = true
-	d.enableModules = append(d.enableModules, modstgs3.Name, modsloads3.Name)
-	return d
-}
-
-func (d *Compose) WithMinIOBucket(bucket string) *Compose {
-	d.withMinIO = true
-	d.enableModules = append(d.enableModules, modstgs3.Name, modsloads3.Name)
-	d.withMinIOBucketName = bucket
-	return d
+	return &Compose{enableModules: []string{}, weaviateEnvs: make(map[string]string), withBackendS3Buckets: make(map[string]string)}
```

Comment:
I think that `withBackendS3Buckets` is not needed here, bc those buckets are only passed to MinIO container

Reply:
ohhh, it actually is needed, ok :)